lib/checks/forms/submit-button.js

```diff
@@ -0,0 +1,13 @@
+if (
+	node.querySelector(
+		'input[type="submit"]:not([disabled]), img[type="submit"]:not([disabled]), button[type="submit"]:not([disabled])'
+	)
+) {
+	return true;
+}
+
+if (!node.querySelectorAll(':not(textarea)').length) {
+	return false;
```

Comment:
I think Dylan has a valid point here:
https://github.com/dequelabs/axe-core/issues/529#issuecomment-424114684

There's always the possibility that someone scripted up a link as the submit button, or any number of other things. You can simply take out this if statement to solve for that.